src/utils.js

```diff
@@ -126,14 +126,17 @@ export function getVariable(context, name) {
  *
  */
 export function gettext(str) {
-  return str;
+  let locale = getConfig().argv.locale;
+  let i18ndata = require(`./locale/${locale}/messages.js`)
+  let jed = new Jed(i18ndata);
+  return jed.gettext(str);
 }
```

Comment:
~~I think I would put `i18ndata` (as `I18N_DATA` or so) at the global level (just like `SOURCE_MAP_RE` above) for various reasons:~~

**Update:** So I talked to @rpl and we came up that the least magic implementation indeed would be to just rely on `process.ENV` for now, as you originally did and ignore the fact that we might want a cli option.

The problems around the cli option are quite huge and resulted in me scratching up https://gist.github.com/EnTeQuAk/001fb42cab37b7f20a4d9f6df8030a63 which is something we might not want to do but it's been fun.

So, let's summarize my finalized thoughts on this:

 * relying on `process.ENV` is simple and allows `Jed` to be globally initialized in `utils.js` from the very beginning
 * No need to overwrite the currently used locale, anywhere. We can just use what's defined by the environment
 * re-using the environment variables isn't a blocker to make it useful for addons-server, it's just 1-2 lines more code which is reasonable and doesn't justify any use of magic

The problems with the cli argument are:

 * We'd need to make our strings that get returned by `gettext` be translated on **first access** instead of directly during the `gettext` call, otherwise they would always be English 'cause that's what we initialize by default
 * That would mean the use of `Proxy` and other laziness methodologies which might not be worth it.

Does that make sense?